.devcontainer/Dockerfile

```diff
@@ -0,0 +1 @@
+FROM quay.io/redhat-pipeline-service/devenv:main
```

Comment:
You can manually upload this image to the redhat-pipeline-service repo in quay because,
1. Others can test it.
2. You need to manually make it public. Otherwise, even if you wait for the PR merge to upload the image, it won't make it public and nobody can still use it.

Reply:
Done